woocommerce/woocommerce-android
# feat: add tracks events for product bulk update

Closes: #8004


### Description

This PR adds tracking events to the newly added feature of products bulk update.

### Testing instructions

Not needed as there are unit tests. Alternatively, you can make sure by running a LogCat and see if related events are tracked.
